specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json

```diff
@@ -350,6 +347,9 @@
           }
         ],
         "responses": {
+          "200": {
+            "description": "The retrieval of the operation was successful."
```

Comment:
nit: Either "The operation was successful" Or more specific "The Run Operation was successful"